src/five/formlib/tests/test_formlib.py

```diff
@@ -68,8 +68,7 @@ def http_request(url, form_parts=None, body=None, auth=None):
     """perform HTTP request from given parameters.
 
     The primary purpose of this auxiliary function is to compute
-    the `Content-Length` header. While Python 2's `cgi` module
-    seems resilient to a wrong `Content-Length`, Python 3's `cgi` fails
+    the `Content-Length` header. Python 3's `cgi` fails
```

Comment:
`Zope` no longer uses `cgi`. However, if a `Content-Length` header is present, it must still be correct.